app-server/app/src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level=${APP_SERVER_LOG_LEVEL:-info}>
```

Comment:
@notthatbreezy I think we can just add that environment variable to the `s3` development config file and not have to do anything else